omplConfig.cmake.in

```diff
@@ -1,63 +1,13 @@
-# CMake OMPL module
+# CMake OMPL config module
 #
-# It defines the following variables:
-# OMPL_FOUND         - TRUE
-# OMPL_INCLUDE_DIRS  - The OMPL include directories
-# OMPL_LIBRARY_DIRS  - The OMPL library directories
-# OMPL_LIBRARIES     - The OMPL library
-# OMPLAPP_LIBRARIES  - The OMPL.app libraries (if installed)
-# OMPL_VERSION       - The OMPL version in the form <major>.<minor>.<patchlevel>
-# OMPL_MAJOR_VERSION - Major version
-# OMPL_MINOR_VERSION - Minor version
-# OMPL_PATCH_VERSION - Patch version
+# Usage:
+# find_package(ompl @OMPL_VERSION_MAJOR@ REQUIRED)
+# add_library(your_app main.cpp)
+# target_link_libraries(your_app PRIVATE ompl::ompl)
 
 @PACKAGE_INIT@
 
-set(OMPL_VERSION @PROJECT_VERSION@)
-set(OMPL_MAJOR_VERSION @PROJECT_VERSION_MAJOR@)
-set(OMPL_MINOR_VERSION @PROJECT_VERSION_MINOR@)
-set(OMPL_PATCH_VERSION @PROJECT_VERSION_PATCH@)
+include ("${CMAKE_CURRENT_LIST_DIR}/omplExport.cmake" )
+include(CMakeFindDependencyMacro)
+find_dependency(Boost REQUIRED COMPONENTS serialization filesystem system program_options)
```

Comment:
`program_options` is only used in demos inside this repo, so it's not necessary to export it.

see https://github.com/ompl/ompl/pull/1085

Reply:
Done in c13e0e2f36257b26648586961688422ba19841cd